packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -79,8 +80,11 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
           isOpen: true
         });
         // Add a keyUp handler for all key up events when the dialog is open
-        if (newProps.dragOptions) {
+        if (newProps.dragOptions && !this.state.hasRegisteredKeyUp) {
```

Comment:
Should `hasRegisteredKeyUp` be state, could it just be a private member?

Reply:
Good call, I updated it